PROCESS.md

```diff
@@ -0,0 +1,109 @@
+   TSC meeting where consensus can be reached around the proposal's relevance 
+   and approach. TSC meetings are public and meeting notes are posted to the 
+   [OCIO repo](https://github.com/AcademySoftwareFoundation/OpenColorIO/tree/master/ASWF/meetings/tsc) 
+   within days of a meeting for a permanent record of decisions.
+2. Following acceptance of the proposal by the TSC, an issue is created in 
+   GitHub to present a design proposal. A period of further community 
+   discussion is encouraged around the proposed changes, captured in issue 
+   comments. Ongoing concerns may continue to be raised in this issue, and 
+   brought to the TSC prior to and during development.
+3. Feature proposal issues are added to the 
+   [Feature Development](https://github.com/AcademySoftwareFoundation/OpenColorIO/projects/3) 
+   project in GitHub by a committer for tracking the status of all accepted 
+   proposals in one place.
+4. During development, work in progress may optionally be shared for feedback 
+   and early review via a 
+   [draft pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests).
+5. Upon completion of feature development, a pull request is created for review 
+   and 
+   [linked to the associated issue]((https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) for review.
+   Linking the issue and PR will result in the issue being closed automatically 
+   upon the PR being merged, and facilitate tracking of the proposed feature in
+   [Feature Development](https://github.com/AcademySoftwareFoundation/OpenColorIO/projects/3). 
+   See [CONTRIBUTING.md](CONTRIBUTING.md) for an overview of commit signing 
+   requirements and a guide on submitting a pull request.
+6. The pull request is merged after being approved as documented in the 
+   [Required Approvals](#Required-Approvals) section below.
+
+**NOTE**: Software design is often an iterative process with development. While 
+a design proposal is required, it is understood that development of the 
+proposed feature may begin prior to opening a design proposal issue in GitHub. It 
+is however highly encouraged to propose the design early in the interest of 
+respecting the contributor's time and effort. This avoids getting too far into 
+development without community involvement, which may lead to redesign and 
+extensive rework before changes can be approved.
+
+## Required Approvals
+
+Modifications of the contents of the OpenColorIO repository are made on a
+collaborative basis. Anyone with a GitHub account may propose a modification via
+pull request and it will be considered by the project Committers.
+
+Pull requests must meet a minimum number of Committer approvals prior to being
+merged. Rather than having a hard rule for all PRs, the requirement is based on
+the complexity and risk of the proposed changes, factoring in the length of
```

Comment:
I interpreted your previous third comment as suggesting this should be removed:

> The current GitHub protection settings require two committer approvals before anything may be merged.

Reply:
Nevermind, I misunderstood this comment. But the section you are suggesting adding "Large or significant changes" to doesn't carry any condition other than having two approvers, so what would be the distinction between that and the proposed small change rule?

Reply:
The distinction is that approvers for large changes cannot have the same affiliation, whereas they could for small changes.  For example, if it's a bug fix authored by Patrick, I should be allowed to be an approver even before the two weeks has elapsed.

Reply:
Thanks to clearly highlight the critical distinction between small pull requests (i.e. bug fix, etc.) and larges ones (i.e. new feature, etc.). The approval process must authorize a fast-track approach for small pull requests to avoid blocking contributors (even if creator and approver have the same affiliation as described by Doug).  OCIOv2 starts its stabilization phase where bug fixes could pop at anytime needing a fast-track merge (i.e. within few days).

Reply:
I moved the general two approval description to its own bullet point, followed by two others with the rules based on change scope. I also added additional notes around minimum PR lifetime and giving notice before 2 week merges.